app/presenters/task_list/start_page_collection.rb

```diff
@@ -0,0 +1,94 @@
+module TaskList
+  class StartPageCollection < Collection
+    NEW_RECORD = "00000000-0000-0000-0000-000000000000".freeze
+
+    # Template:
+    #
+    # SECTIONS = {
+    #   task_list_section: {
+    #    task_list_item_1 (name after step): displayed?,
+    #    task_list_item_2 (name after step): displayed?,
+    #    ....
+    #    subsections: {
+    #      subsection_name: {
+    #        sub_task_list_item_1 (name after step): displayed?
+    #      }
+    #    }
+    #   }
+    # }
+    #
+    # Rules: name task list items after the [first] step in its flow.
+    #
+    # e.g.
+    #  - Steps::ProviderStart::ApplicantsStep --> applicants
+    #  - Steps::ProviderStart::AboutFinancialMeansStep --> about_financial_means
+    #
+    # Body overide:
+    #
+    # If the section's task list cannot be shown and requires text to be displayed instead then the
+    # body_override task key should be added. It's lambda should return the message to be displayed
+    # only if it needs to be displayed. This will override displaying of the tasks for that section
+    # replacing the tasks with the message body provided.
+    #
+    # e.g. body_override: ->(application) { "Not available yet because..." unless application.client_and_case_details_completed? },
+    #
+
+    SECTIONS = {
+      client_and_case_details: {
+        applicants: true, # Steps::ProviderStart::ApplicantsStep
+        # links_to_another: true, # TODO
+        proceedings_types: true, # Steps::ProviderStart::ProceedingsTypesStep,
+        check_provider_answers: true, # Steps::ProviderStart::CheckProviderAnswersStep
+        check_benefits: true, # Steps::ProviderStart::CheckBenefitsStep
+        # ^^ DWP Result check - THIS SECTION IS CONDITIONAL. it should perhaps not exist as depending on answers to previous
+        # questions it would be different AND if answers change (*such as age) it would need to change the forward path
+        # and even remove/soft-delete/discard existing data (e.g. means test answers).
+        # [
+        #   "check_benefits", # Steps::ProviderStart::CheckBenefitsStep
+        #   "confirm_non_means_tested_application", # Steps::ProviderStart::ConfirmNonMeansTestedApplicationStep
+        #   "no_national_insurance_numbers", # Steps::ProviderStart::NoNationalInsuranceNumbersStep
+        # ],
```

Comment:
@RoseSAK since you are working on the DWP check task list item it is worth looking at this spike work in which i had made a very basic start. In particular the fact that the task list item may need to be conditional.